lark/visitors.py

```diff
@@ -52,7 +55,7 @@ def __class_getitem__(cls, _):
         return cls
 
 
-class Transformer(_Decoratable, ABC, Generic[_T]):
+class Transformer(_Decoratable, ABC, Generic[_Return_T, _Leaf_T]):
```

Comment:
Wouldn't it make more sense if `_Return_T` was the last param? i.e. following an (in, out) pattern.

Reply:
This was brought up earlier, so I've made that change now.